scripts/find_unused_locales.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
We assume grep below, so might as well assume unix env, and start the file with:
```
#!/usr/bin/env node
```
then remove the .js extension and `chmod +x` it, then can execute it directly...